crates/zed/Cargo.toml

```diff
@@ -143,6 +143,7 @@ tree-sitter-nu.workspace = true
 tree-sitter-vue.workspace = true
 tree-sitter-uiua.workspace = true
 tree-sitter-zig.workspace = true
+tree-sitter-proto.workspace = true
```

Comment:
Ditto here, let’s try to keep these in alphabetical order.

Reply:
You got it. I initially did that everywhere but I think I saw the entries here weren't sorted so I thought maybe you guys were just appending. I'll get that fixed.

Reply:
I think they're _mostly_ sorted at the moment, but it's possible that some aren't entirely in order 😄